From: Li kunyu <kunyu@nfschina.com>
Subject: kernel: relay: remove unnecessary NULL values from relay_open_buf
Date: Fri, 14 Jul 2023 07:44:59 +0800

buf is assigned first, so it does not need to initialize the assignment.

Link: https://lkml.kernel.org/r/20230713234459.2908-1-kunyu@nfschina.com
Signed-off-by: Li kunyu <kunyu@nfschina.com>
Reviewed-by: Andrew Morton <akpm@linux-foudation.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 kernel/relay.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/kernel/relay.c~kernel-relay-remove-unnecessary-null-values-from-relay_open_buf
+++ a/kernel/relay.c
@@ -375,7 +375,7 @@ static struct dentry *relay_create_buf_f
  */
 static struct rchan_buf *relay_open_buf(struct rchan *chan, unsigned int cpu)
 {
- 	struct rchan_buf *buf = NULL;
+	struct rchan_buf *buf;
 	struct dentry *dentry;
 
  	if (chan->is_global)
_
